config/env/default.js

```diff
@@ -26,6 +26,10 @@ module.exports = {
   domain: process.env.DOMAIN || 'localhost:3000',
   supportEmail: 'support@trustroots.org', // TO-address for support requests
   profileMinimumLength: 140, // Require User.profile.description to be >=140 chars to send messages
+  // A message below longMessageMinimumLength will be 'short' in influxdb, otherwise 'long'
+  // TODO choose an expressive value
+  // Used in ./modules/messages/server/services/message-to-influx.server.service
+  longMessageMinimumLength: 150,
```

Comment:
Influxdb config block would be only at `default` and `local` config files, so to change that value in production, one would edit only `local.js`. Only problem would be that if we'd add anything to `default.js`, it would need to be manually copied over to `local.js` as well (because config blocks are not _merged_ but overwritten).

I was first thinking something along the lines of this:

```javascript
influxdb: {
  limits: {
    // Messages shorter than this will be tagged 'short' in influxdb, otherwise 'long'
    longMessageMinimumLength: 150,
  },
  ...
}
```

But on second thought we might add limits which would be used by both application logic and influxdb. E.g. `profileMinimumLength` (already in configs) would probably be used by Influxdb as well. Might be we would add `longProfileMinimumLength` and other stuff like this later on.

So I'd say let's go with this:
```javascript
influxdb: {
  ...
},
limits: {
   // Messages shorter than this will be tagged 'short' in influxdb, otherwise 'long'
   longMessageMinimumLength: 150
},
``` 
...and after merging this PR we can move `profileMinimumLength` there, too.

Thoughts?